### ISSUE TITLE
Snyk vulnerabilities

### ISSUE THREAD

**Issue by chosak** [cfpb/college-costs#325]
The `npm test` command fails because its subcommand `snyk test` fails with several high severity vulnerabilities:

> Tested 586 dependencies for known vulnerabilities, found 20 vulnerabilities, 69 vulnerable paths.

To reproduce, first log in to [Snyk](https://snyk.io/) using `snyk auth` and then run `snyk test` from the command line. Alternatively, you can browse the vulnerabilities through the [Snyk web UI](https://app.snyk.io/org/cfpb) (requires membership in the "cfpb" organization).

The high severity vulnerabilities seem to come from `student-debt-calc@2.6.3`, which is also a CFPB package from https://github.com/cfpb/student-debt-calculator. Correcting the vulnerabilities here may require a new version release of that package that corrects its vulnerabilities.

**Comment by chosak** [comment]
Now that @mistergone has merged #328, some of these vulnerabilities have been fixed, but one still remains when running `npm test`:

```
✗ Medium severity vulnerability found in jquery
  Description: Cross-site Scripting (XSS)
  Info: https://snyk.io/vuln/npm:jquery:20150627
  Introduced through: jquery@1.11.3, jquery.easing@1.3.2
  From: jquery@1.11.3
  From: jquery.easing@1.3.2 > jquery@1.11.3
  Remediation:
    Your dependencies are out of date, otherwise you would be using a newer version of jquery. 
    Try deleting node_modules, reinstalling and running `snyk test` again. If the problem persists, one of your dependencies may be bundling outdated modules.

Organisation:    chosak
Package manager: npm
Target file:     package.json
Open source:     no
Project path:    /Projects/college-costs
Local Snyk policy: found

Tested 327 dependencies for known vulnerabilities, found 1 vulnerability, 2 vulnerable paths.

Run `snyk wizard` to address these issues.

npm ERR! Test failed.  See above for more details.
```

One option would be to update this repo's [.snyk file](https://github.com/cfpb/college-costs/blob/master/.snyk) to ignore this jQuery vulnerability. We could also change the `npm test` command so that it runs `snyk test --severity-threshold=high`.

**Comment by chosak** [comment]
Closed by #332:

```sh
$ npm run snyk

> college-costs@ snyk /Users/chosaka/Projects/college-costs
> snyk test


Testing /Users/chosaka/Projects/college-costs...

Organisation:    chosak
Package manager: npm
Target file:     package.json
Open source:     no
Project path:    /Users/chosaka/Projects/college-costs
Local Snyk policy: found

✓ Tested 327 dependencies for known vulnerabilities, no vulnerable paths found.
```
